### ISSUE TITLE
Device configure

### ISSUE THREAD

**Issue by disaster123** [iuriaranda/venus-docker#5]
Hi,

thanks for all your hard work. I got all containers up and running and have configures the main basic stuff through VEConfigure.

But later on in all documentation they say i snhould connect via http to Venus OS to make the relevant settings for an ESS setup. What do i miss? The docker images do not seem to provide a http service...

Greets,
Stefan

**Comment by iuriaranda** [comment]
Indeed, as far as I know, none of the packages I dockerized in this repo provide an http service that you can use with VEConfigure. You could check the [Victron apt repository](https://updates.victronenergy.com/feeds/venus/release/packages/debian-buster/dists/buster/main/binary-armhf/Packages) to see if there's a package that fits the description of what you are looking for. Although it could be that Victron not even publishes that as a package and it's something proprietary only available within the Venus OS images.
